tests/usm/usm_atomic_access_atomic64_fp64.cpp

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provide verification to atomic access for USM allocations that underlying
+//  type size equal than 64 byte.
+//
+*******************************************************************************/
+
+#include "usm_atomic_access.h"
+
+#define TEST_NAME usm_atomic_access_atomic64_fp64
+
+namespace TEST_NAMESPACE {
+using namespace sycl_cts;
+
+/** Test instance
+ */
+class TEST_NAME : public sycl_cts::util::test_base {
+ public:
+  /** return information about this test
+   */
+  void get_info(test_base::info &out) const override {
+    set_test_info(out, TOSTRING(TEST_NAME), TEST_FILE);
+  }
+
+  /** execute the test
+   */
+  void run(util::logger &log) override {
```

Comment:
Please, use Catch2 testing framework instead.

Reply:
Do you have an example of a test-case that uses Catch2 for reference?

Reply:
https://github.com/KhronosGroup/SYCL-CTS/blob/SYCL-2020/tests/event/event.cpp
https://github.com/KhronosGroup/SYCL-CTS/blob/SYCL-2020/docs/README.adoc

Reply:
Great. Thanks a lot!